tests/blockchain/eth/contracts/integration/test_intercontract_integration.py

```diff
@@ -118,12 +241,13 @@ def user_escrow_proxy(testerchain, token, escrow, policy_manager):
 
 
 @pytest.mark.slow
-def test_all(testerchain, token, escrow, policy_manager, user_escrow_proxy):
+def test_all(testerchain, token, escrow, policy_manager, adjudicator, user_escrow_proxy):
     # Travel to the start of the next period to prevent problems with unexpected overflow first period
     testerchain.time_travel(hours=1)
 
     escrow, escrow_dispatcher = escrow
     policy_manager, policy_manager_dispatcher = policy_manager
+    adjudicator, adjudicator_dispatcher = adjudicator
```

Comment:
Lines like this drive me crazy.  What was `adjudicator` if the first member is `adjudicator`?

Reply:
Agreed, not the best naming
first `adjudicator`, `escrow`, `policy_manager` are tuples: (contract_abi, dispatcher_abi)